lib/daos/userSubjectsDao.js

```diff
@@ -0,0 +1,21 @@
+import { user_subjects as userSubjects } from 'models';
+import sequelize from 'sequelize';
+const attributes = ['id', 'userId', 'subjectId'];
+export const findAllUserSubjects = async (where, page, limit, include = []) => {
+    const totalCount = await userSubjects.count({ where });
+    const allUserSubjects = await userSubjects.findAll({
+        where,
+        order: sequelize.literal('id ASC'),
+        limit,
+        offset: (page - 1) * limit,
+        include
+    });
+
+    return {
```

Comment:
it would make sense for the dao layer to spit out a consistent format. 

For example since we already know that the hapi-pagination plugin requires results and totalCount we should return that from the daos that are returning an array. 

results, totalCount

In this way you don't need to reassign in the route